docs/user/BuildLocally.md

```diff
@@ -0,0 +1,46 @@
+# Build from sources locally
+
+## Prerequisites
+
+### OpenROAD
+
+Dependencies for OpenROAD app are documented in the script below.
+During initial setup or if you have a new machine, run this script:
+
+``` shell
+# either run as root or use sudo
+./etc/DependencyInstaller.sh -dev
+```
+
```

Comment:
In ORFS we should list OR as a dependency and link to the OR-docs for how to install it.